### PR TITLE
Update e2e-crc tests

### PR DESCRIPTION
**Pull request by praveenkumar**
This patch will use the ci script which present
in the snc repo instead of putting again everything here.

### CODE REVIEW

File: ci-operator/step-registry/openshift/e2e/gcp/crc/test/openshift-e2e-gcp-crc-test-commands.sh [hunk 1]
```diff
@@ -26,67 +26,21 @@ gcloud --quiet config set project "${GOOGLE_PROJECT_ID}"
...
-  export OPENSHIFT_PULL_SECRET_PATH="${HOME}"/pull-secret
-  export OPENSHIFT_VERSION="$(curl -L "${MIRROR}"/latest/release.txt | sed -n 's/^ *Version: *//p')"
-  BUNDLE_VERSION=${OPENSHIFT_VERSION}
-
   # clone the snc repo
   git clone https://github.com/code-ready/snc.git
   pushd snc
+  set -e
+  export OPENSHIFT_PULL_SECRET_PATH="${HOME}"/pull-secret
   ./ci.sh
```
Comment (cfergeau) [inline ci-operator/step-registry/openshift/e2e/gcp/crc/test/openshift-e2e-gcp-crc-test-commands.sh:42]: I'm confused by the log, isn't this already using the `ci.sh` script even before this commit?

Comment (cfergeau) [reply ci-operator/step-registry/openshift/e2e/gcp/crc/test/openshift-e2e-gcp-crc-test-commands.sh:42]: Ok, this code has been moved to snc as part of https://github.com/code-ready/snc/commit/e6cbd23ad255c862597fffdb021846ba70a58aa4

**Review by cfergeau — CHANGES_REQUESTED** [review]
/approve if the commit log is slightly reworded.

**Comment by praveenkumar** [comment]
/hold

Till CI is green

**Review by cfergeau — APPROVED** [review]

**Pull request merged**